board-viewer.html

```diff
@@ -447,6 +447,7 @@
 
   _onEscPressed: function(event) {
     this.selected = [];
+    this._abortZoomToFit();
```

Comment:
I'm adding a reset of the scale to 'fitScale' size and centering the board when the 'esc' key is pressed together with the clean up of the selected array. In my opinion the 'esc' key press functionality works fine reseting the scale and cleaning the selected array. But also I think that could makes sense if the 'esc' only reset the scale and center the board when the 'ZoomedFit' is true, and press 'esc' other time to deselect the filters... what do you think @jketreno about double 'esc' press or a single 'esc' press?

Reply:
ESC should only clear the active selection; it shouldn't re-scale or re-center.

The behavior is intended to mimic item selection in other contexts/UI patterns.  In editors, ESC is frequently used to abort the selection process (our intended behaviour), to stop an action, or dismiss a dialog  -- it rarely changes the UI itself.

Since your code is adding the center-focus icon, I think the UI hinting is sufficient for the user to know they can re-center using that icon.

If you wanted to add a HOME key trigger to recenter the board, that would seem appropriate.

For all our keyboard handlers, it looks like we need to check that focus is in our window before we consume the event... I just noticed we steal the ESC from the documentation pane :)

Reply:
I'll manually merge the code, removing the abortZoomToFit when ESC is pressed. I will also file a new issue for us to verify the target of the event for keystrokes.